synapse/cortex.py

```diff
@@ -276,7 +276,20 @@ async def _reqDefLayerAllowed(self, perms):
         view = self.cell.getView()
         self.user.confirm(perms, gateiden=view.wlyr.iden)
 
-    async def addFeedData(self, items, *, viewiden=None):
+    async def importStormMeta(self, meta, extmodel=False, viewiden=None):
```

Comment:
should switch viewiden to a keyword-only arg
```suggestion
    async def importStormMeta(self, meta, *, viewiden=None):
```